_modules/migration.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+
+const dir        = require("./dir");
+const dotnetPath = require("./dotnet-path");
+const echo       = require("./echo");
+const path       = require("path");
+const shell      = require("shelljs");
+const upath      = require("upath");
+
+/**************************************************************************************************
+ * Constants
+ **************************************************************************************************/
+
+// #region Constants
+
+const MIGRATION_MODE = {
+    ADD:    "ADD",
+    DELETE: "DELETE",
+    RUN:    "RUN",
+};
+
+const MIGRATION_COMMAND = {
+    [MIGRATION_MODE.ADD]:    " migrations add ",
```

Comment:
While it may seem trivial now, recommend not baking in spaces or other delimiters into the const when possible. Makes it more reusable. Instead, below add the space padding in the method that needs it